.gitignore

```diff
@@ -13,3 +13,7 @@
 
 bin/
 .gobincache/
+
+# IDE configs
+.idea/
+.vscode/
```

Comment:
It's convention to leave a blank new line at the end of every file, please make sure your IDE is set up to do this (this is the red marker on this line that shows it's missing)